src/components/Bank.js

```diff
@@ -33,6 +37,16 @@ function hasExchange(sell, buy) {
   return true;
 }
 
+function calculateTaxes(price) {
```

Comment:
@jlgasparrini 
could be possible extract this into `lib/utils.js` and we could add some tests `lib/utils.test.js`